models/safe/arbitrum/safe_arbitrum_eth_transfers.sql

```diff
@@ -0,0 +1,61 @@
+{{ 
+    config(
+        materialized='incremental',
+        alias='eth_transfers',
+        partition_by = ['block_date'],
+        unique_key = ['block_date', 'tx_hash', 'trace_address', 'amount_raw'],
```

Comment:
Hi @tschubotz , do you think `amount_raw` is necessary to be part of unique_key?

Reply:
Thanks for review.
nope, that's a silly copy paste mistake. Just removed it. Also reused your `project_start_date` variable.

Will replicate both changes to all related PRs as well.

Reply:
Actually, turns out `amount_raw` is required, however the `address` column is way better suited for this, conceptually. I changed it, tests just passed.